features/font-variant.yml

```
@@ -0,0 +1,16 @@
+name: font-variant
+description: The `font-variant` CSS property is a shorthand for all font variant subproperties.
```

I think we have the room to pull this off:

```suggestion
description: The `font-variant` CSS property is a shorthand for `font-variant-alternates`, `font-variant-caps`, `font-variant-east-asian`, `font-variant-emoji`, `font-variant-ligatures`, `font-variant-numeric`, and `font-variant-position`.
```